pkg/api/tmpl/chi-interface.tmpl

```diff
@@ -0,0 +1,7 @@
+// ServerInterface represents all server handlers.
+type ServerInterface interface {
+{{range .}}{{.SummaryAsComment }}{{$opid := .OperationId}}
+// ({{.Method}} {{.Path}}) 
+{{.OperationId}}(w http.ResponseWriter, r *http.Request{{if .HasBody}}{{range .Bodies}}{{if and .Default (eq .NameTag "JSON")}}, body {{$opid}}{{.NameTag}}RequestBody {{end}}{{end}}{{end}}{{genParamArgs .PathParams}}{{if .RequiresParamObject}}, params {{.OperationId}}Params{{end}})
```

Comment:
Dumb question: Is there a way to break this line and add indentation so it's somewhat readable?

Reply:
Oh and also - Great job with the templates!!

Reply:
added some new-lines, let me know if it is better

Reply:
added some new lines.